File: Exercises/101-109 - Small Problems/easy_2/when_will_i_retire.rb
# Build a program that displays when the user will retire and how many years she has to work till retirement.

puts 'What is your age?'
age = gets.to_i

puts 'At what age would you like to retire?'
retire = gets.to_i

years_left = retire - age
current_year = Time.now.year
retirement_year = current_year + years_left

puts "It's #{current_year}. You will retire in #{retirement_year}."
puts "You have only #{years_left} years of work to go!"

File: Exercises/210 - Small Problems/easy-3/when-will-i-retire.js
/*
Build a program that logs when the user will retire and how many years the user has to work till retirement.

Example:

What is your age? 30
At what age would you like to retire? 70

It's 2016. You will retire in 2056.
You have only 40 years of work to go!
*/

var age = Number(prompt('What is your age?'));
var retire = Number(prompt('At what age would you like to retire?'));
var today = new Date();
var year = today.getFullYear();

console.log("It's " + year + ' You will retire in ' + (year + retire - age));
console.log('You only have ' + (retire - age) + ' years of work to go!');